src/sections/new-collection/collection-form/browse-search-facets-section/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { useTranslation } from 'react-i18next'
```

Comment:
I'm curious why this file and others are named index.tsx rather than the component name?  Is there a convention for deciding in what situations to use one rather than the other?

Reply:
This is just a convention I follow to avoid but I can change it if you want. When using index as the main file of the folder, when importing it, it will only look like this:
`import { FooComponent } from 'sections/bla/foo-component'`
instead of 
`import { FooComponent } from 'sections/bla/foo-component/FooComponent'`

But if you feel like is confusing I can start working with the component name instead.

Reply:
ok, I think I like having component name in the file, because it's easier when reviewing multiple files, and having multiple file tabs open...instead of looking to see what component is in the index.tsx file, the file name will tell us that.

Reply:
thats a good point, will rename index files from this PR 👍🏼

Reply:
Renamed from index to component name 👍🏼